packages/components/src/ui/styles/system.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Internal dependencies
+ */
+import { createStyleSystem, get as getConfig } from '../create-styles';
+import {
+	config,
+	darkHighContrastModeConfig,
+	darkModeConfig,
+	highContrastModeConfig,
+} from './theme';
+
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @typedef {
+ | 'colorAdmin'
+ | 'colorAdminRgba10'
+ | 'colorAdminRgba20'
+ | 'colorAdminRgba30'
+ | 'colorAdminRgba40'
+ | 'colorAdminRgba50'
+ | 'colorAdminRgba60'
+ | 'colorAdminRgba70'
+ | 'colorAdminRgba80'
+ | 'colorAdminRgba90'
+ | 'colorDestructive'
+ | 'colorDestructiveRgba10'
```

Comment:
@sarayourfriend On one of my dev environments (but curiously, not both), I'm seeing the following list of CSS variables in trunk:

<img width="530" alt="Screenshot 2021-04-12 at 08 46 24" src="https://user-images.githubusercontent.com/1204802/114365231-8daa5d80-9b7a-11eb-9b62-aaf23486bc5b.png">

I can't find any reference to those in the code, though some of the names do seem shared with this list here. Do you know where they are coming from?

My primary concern here is that the block editor, as far as I'm aware, does not use a single of the 500+ CSS variables defined here, and in fact has its own set of CSS variables that diverge slightly from these. Mostly, it's a lot of weight to carry around on every pageload, and will likely cause deprecation headaches if developers start relying on these.

Reply:
Those are coming from the current integration with the new style system via the published packages in the `@wp-g2` namespace. The block editor uses the new FontSizePicker which uses the new style system via the packages, which is why the variables are being added there.

Once this PR and the next to add the `context` system is merged, we'll start to not rely on the published packages which will change the prefix from `wp-g2` to `wp-experimental`, so hopefully no one will rely on them.

Reply:
Thank you, presumably then we can retire most, if not all of them? And/or unify with the existing set of custom properties we have defined.

Reply:
Yup, in https://github.com/WordPress/gutenberg/pull/30630/commits/d1c7e146d93650d0e37537ac22ff9b5af1c2a506 I removed anything that didn’t align with the existing variables.

Reply:
Awesome :heart: